src/main/java/in/projecteka/consentmanager/clients/ClientError.java

```diff
@@ -154,6 +154,10 @@ public static ClientError failedToUpdateTransactionPin() {
         return internalServerError("Failed to update request_id in transaction pin");
     }
 
+    public static ClientError failedToEditTransactionPin() {
+        return internalServerError("Failed to update pin in transaction pin");
```

Comment:
How about `Failed to update transaction pin`?

Reply:
done